editor/src/components/canvas/canvas-strategies/strategies/absolute-reparent-strategy.spec.browser2.tsx

```diff
@@ -1175,9 +1199,11 @@ describe('children-affecting reparent tests', () => {
 function getOpeningTag(type: ContentAffectingType): string {
   switch (type) {
     case 'sizeless-div':
-      return `<div data-uid='children-affecting' data-testid='children-affecting'>`
+      return `<div data-uid='children-affecting' data-testid='children-affecting'><>`
     case 'fragment':
-      return `<React.Fragment data-uid='children-affecting' data-testid='children-affecting'>`
+      return `<React.Fragment data-uid='children-affecting' data-testid='children-affecting'><>`
+    case 'conditional':
+      return `{ true ? ( <>`
```

Comment:
Why are these now also including a (shorthand) fragment?

Reply:
because the test code has two siblings as children of this element, which is possible for divs and fragments, but not for conditional root elements. 

actually the nested fragment is not a bad thing anyways because we uncovered some bugs with nested groups in the last week